cicada/commands/exec_schedule.py

```diff
@@ -242,20 +258,25 @@ def main(schedule_id, dbname=None):
                 zombie.send_signal(signal.SIGTERM)
 
             # Repeatedly attempt to finalize schedule, even if db is unavailable
-            db_connection_made = False
-            while not db_connection_made:
+            db_conn_success = False
+            while not db_conn_success:
```

Comment:
nit: 

can be written in this way also and remove `db_conn_success` : 

```python 
while (True):
        try:
            db_conn = postgres.db_cicada(dbname)
            db_cur = db_conn.cursor()
            break
        except Exception:
            ...
        
```

Reply:
true ... will change this